packages/shared/src/components/cards/CollectionCard/CollectionCard.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { forwardRef, Ref, useState } from 'react';
+import classNames from 'classnames';
+import { Container, PostCardProps } from '../common';
+import FeedItemContainer from '../FeedItemContainer';
+import { CollectionCardHeader } from './CollectionCardHeader';
+import {
+  getPostClassNames,
+  FreeformCardTitle,
+  CardSpace,
+  CardButton,
+} from '../Card';
+import { WelcomePostCardFooter } from '../WelcomePostCardFooter';
+import ActionButtons from '../ActionButtons';
+import PostMetadata from '../PostMetadata';
+
+export const CollectionCard = forwardRef(function CollectionCard(
+  {
+    children,
+    post,
+    domProps = {},
+    onUpvoteClick,
+    onCommentClick,
+    onMenuClick,
+    onShare,
+    onShareClick,
+    openNewTab,
+    onReadArticleClick,
+    onPostClick,
+  }: PostCardProps,
+  ref: Ref<HTMLElement>,
+) {
+  const [headingFull, toggleHeadingFull] = useState(false);
+  const clamp = (() => {
+    if (post.image) {
+      return 'line-clamp-3';
+    }
+
+    return post.contentHtml ? 'line-clamp-4' : 'line-clamp-9';
+  })();
+
+  return (
+    <FeedItemContainer
+      domProps={{
+        ...domProps,
+        className: getPostClassNames(post, domProps.className, 'min-h-card'),
+        onMouseEnter: () => toggleHeadingFull(true),
+        onMouseLeave: () => toggleHeadingFull(false),
```

Comment:
This could just be a hover state no? (group-hover)

Reply:
This is no longer needed for now, since we are no longer loading related sources on hover. But you bring a good point, if we ever circle back to it I will apply your suggestion